product_lot_sequence/models/product.py

```diff
@@ -8,6 +8,9 @@
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
+    disallow_lot_sequence = fields.Boolean(
+        help="Don't use an specific lot sequence for this product"
+    )
```

Comment:
Would be great to have this variable inversed in meaning with a migration script.
Seems more readable and less confusing

```suggestion
    use_specific_lot_sequence = fields.Boolean(
        help="Use an specific lot sequence for this product",
        default=True,
    )
```

Reply:
Done